src/common/wizard/wizard.scss

```diff
@@ -26,3 +26,12 @@
     }
   }
 }
+
+/* Removes the bottom margin when gdpr notes are shown in the bottom of the wizard */
+.vene-form-container {
+  &__without-bottom-margin {
```

Comment:
does this work actually? its interpreted like className="vene-form-container __without-bottom-margin" with space. its like element which also has class

Reply:
Should be working. Tested while developing. This kind of solution was used somewhere else in Vene too.